src/qibolab/serialize.py

```diff
@@ -108,6 +111,11 @@ def dump_qubits(qubits: QubitMap, pairs: QubitPairMap, couplers: CouplerMap = No
     characterization = {
         "single_qubit": {q: qubit.characterization for q, qubit in qubits.items()},
     }
+    for q in qubits:
+        kernel_path = characterization["single_qubit"][q].pop("kernel_path")
+        if kernel_path is not None:
+            characterization["single_qubit"][q]["kernel_path"] = kernel_path.name
```

Comment:
```suggestion
        qubit = characterization["single_qubit"][q]
        kernel_path = qubit["kernel_path"]
        if kernel_path is not None:
            qubit["kernel_path"] = kernel_path.name
```

I haven't found any better way (here the ideal solution would be Pandas' `.apply()` method), but I suggest avoiding `.pop()`, because it's better to always keep the same format. I.e. if some values are empty, better to make them explicitly empty (`None`).